### ISSUE TITLE
Abandoned?

### ISSUE THREAD

**Issue by hamishtaplin** [csswizardry/csswizardry-grids#47]
I really like this system due to the BEM approach but it's not been updated in 6 months and has some issues outstanding, not least compatibility with ems.

Are there plans to further develop it? Would be a shame to let it die as it's unique as far as grids are concerned, imo.

**Comment by nilswindisch** [comment]
Harry seems to be busy with other things... and understandably so. But I like to see csswizardry-grids evolve as much as you do.

**Comment by aristretto** [comment]
@hamishtaplin @nilswindisch judging by his recent commit history (https://github.com/inuitcss), he's hard at work improving inuitcss, which probably means updates will trickle down to his other frameworks such as this one, and typecsset.  at least I hope so :P

**Comment by nilswindisch** [comment]
@aristretto yes, i'm aware of that and the moment @csswizardry puts up his announced Kickstarter campaign for inuit i will contribute to it.
